.changeset/curvy-tigers-cross.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-notifications-backend': patch
+'@backstage/plugin-notifications': patch
+---
+
+all notifications can be marked and filtered by severity critical, high, normal or low, the default is 'normal'
```

Comment:
Should `All` be capitalized as these will be visible in the release notes, not major thing

Reply:
Yeah, the majority of changesets starts with a capital letter.